### ISSUE TITLE
Fix button icon alignment issues

### ISSUE THREAD

**Issue by designmatty** [raster-foundry/blasterjs#231]
See the discussion here: 
- https://azavea.slack.com/archives/CC26JAMS5/p1568062575046700
- https://azavea.slack.com/archives/CC26JAMS5/p1568083730052900

Issues:
- Icons are misaligned vertically
- using `iconBefore` makes the button height slightly larger despite the icon itself being smaller than the text

tldr: 
This only happens when using `iconBefore`
![image](https://user-images.githubusercontent.com/1928955/64885550-77e95500-d632-11e9-81b9-34a290a75d13.png)
![weird-button-issue](https://user-images.githubusercontent.com/1928955/64885772-eb8b6200-d632-11e9-9c64-a95b9355045c.gif)

**Comment by jfrankl** [comment]
I wanted to nudge this issue. I just spent about an hour trying to fix this without touching the component and still can't get these aligned. Would be great if you could address this.

Part of the issue is that:

- My application has other buttons that contain SVG elements that aren't added through `iconBefore` and `iconAfter`
- My application has other buttons that contain `span` elements

I can't find a unique selector that would only target icons added using `iconBefore` and `iconAfter`.

**Comment by jfrankl** [comment]
This block of code in the button theme fixed the issue across my application.

```
display: inline-flex;
& > span {
    display: flex;
    align-items: center;
}
```

If I were modifying the component, I would apply `display: flex` on a wrapper element rather than the button element itself.

**Comment by designmatty** [comment]
fixed with #257
